packages/redis/test/Adaptor.test.js

```diff
@@ -0,0 +1,45 @@
+import { expect } from 'chai';
+
+import { get, set, setMockClient } from '../src';
+
+describe('get', () => {
+  it('should get the string value of a key', async () => {
+    const client = {
+      get: () => {
+        return Promise.resolve('ok');
+      },
+    };
+    setMockClient(client);
+
+    const state = {};
+    await get('key')(state).then(state => {
+      expect(state.data).to.eql('ok');
+    });
+  });
+
+  it('should return null if key does not exist', async () => {
+    const state = {};
+    const client = {
+      get: () => Promise.resolve(null),
+    };
+    setMockClient(client);
+
+    await get('not-found')(state).then(state => {
```

Comment:
Never never never `await` and `then` in the same statement (rarely should you ever do it in the same function even)

The pattern you've been using in Salesforce lately is bad. It was fine in the Salesforce adaptor because it matches the other tests (which are scheduled to be overhauled anyway). But here it's just bad practice.

Use async/await instead:
```
const result = await get('not-found')(state)
expect(result.data).to.eql(null);
```